.bitmap

```diff
@@ -1056,6 +1062,54 @@
         "mainFile": "index.ts",
         "rootDir": "scopes/code/ui/code-view"
     },
+    "ui/component-compare": {
+        "scope": "",
+        "version": "",
+        "mainFile": "index.ts",
+        "rootDir": "scopes/component/ui/component-compare"
+    },
+    "ui/component-compare-aspects": {
+        "scope": "",
+        "version": "",
+        "mainFile": "index.ts",
+        "rootDir": "scopes/component/ui/component-compare-aspects"
+    },
+    "ui/component-compare-code": {
+        "scope": "",
+        "version": "",
+        "mainFile": "index.ts",
```

Comment:
add compare folder under component/ui and move all the compare ui components 
drop the `component-` from the folder name
=> `ui/compare/code`
applies to all components in the components namespace

Reply:
done